app/src/main/java/com/hedvig/app/SplashActivity.kt

```diff
@@ -85,6 +107,34 @@ class SplashActivity : BaseActivity() {
         } ?: startDefaultActivity(loginStatus)
     }
 
+    private fun startBirthdayActivity(){
+        startActivity(Intent(this, BirthdayActivity::class.java))
+    }
+
+
+    private fun hasBenCelebrated(): Boolean {
+        val pref = applicationContext.getSharedPreferences("MyPref", 0) // 0 - for private mode
+
+        return pref.getBoolean("celebrated",false)
+    }
+
+    private fun setHasBenCelebrated() {
+        val socialSecurityNumber = "199701174617"
```

Comment:
This value probably needs to be loaded from the ViewModel, but I guess this is just some WIP-code 😉